core/src/main/java/org/ehcache/core/EhcacheManager.java

```diff
@@ -679,7 +679,7 @@ public void destroyCache(final String alias) throws CachePersistenceException {
     }
 
     try {
-      removeAndCloseWithoutNotice(alias);
+      removeCache(alias);
```

Comment:
Not a chance this goes in without tests 😉 
And it needs daggy fix against `release/3.2`, `release/3.3` and `master`